### ISSUE TITLE
Youtube Import Error

### ISSUE THREAD

**Issue by saintcd** [SilentSys/SLAM#134]
when i try to import the app say "The Conversion of Type 'YoutubeParseException' in type 'String' is not valid" and i don't know how to fix it

**Comment by allthetime** [comment]
@lcssssss there's nothing you can do without code. I've noticed something weird where certain videos will work and others won't. I don't know if its true, but it seems to be popular/copyrighted videos that don't work and random videos that do right now. I'm doing some work/debugging on it over the next couple weeks that i hope will work. Is anyone else working on this?

**Comment by JesWhite** [comment]
yea there is.

> System.Collections.Generic.KeyNotFoundException: Verilen anahtar sözlükte yoktu.
   konum: VideoLibrary.Helpers.Query.get_Item(String key)
   konum: VideoLibrary.YouTube.Unscramble(String queryString)
   konum: System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   konum: VideoLibrary.YouTube.<ParseVideos>d__6.MoveNext()
   konum: System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   konum: VideoLibrary.ServiceBase`1.VideoSelector(IEnumerable`1 videos)
   konum: VideoLibrary.ServiceBase`1.<GetVideoAsync>d__6.MoveNext()
--- Özel durumun oluşturulduğu önceki konumdan başlayan yığın izlemesinin sonu ---
   konum: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   konum: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   konum: VideoLibrary.ServiceBase`1.<GetVideoAsync>d__5.MoveNext()
--- Özel durumun oluşturulduğu önceki konumdan başlayan yığın izlemesinin sonu ---
   konum: System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   konum: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   konum: VideoLibrary.ServiceBase`1.GetVideo(String videoUri)
   konum: SLAM.YTImport.DownloadWorker_DoWork(Object sender, DoWorkEventArgs e) G:\Projects\source-live-audio-mixer\SLAM\YTImport.vb içinde: satır 46

>

**Comment by megaroeny** [comment]
Experiencing this as well. Tried this video: https://www.youtube.com/watch?v=uzQ9kxs6RL8
![image](https://user-images.githubusercontent.com/1559447/84343871-57c1d480-ab77-11ea-91f6-b9e67cd36067.png)

**Comment by aryan-553** [comment]
@megaroeny @Icarussz  same error. Did you find any solution??

**Comment by megaroeny** [comment]
> @megaroeny @Icarussz same error. Did you find any solution??

Nope, I'm not a software engineer, sorry 😓 I'm not playing CS anymore either.

**Comment by sheharyaar** [comment]
@BoomBookTR @megaroeny @aryan-553 
I have fixed the issue and created another repo as the current moderators are not active. Here is the link to my repo : https://github.com/sheharyaar/SLAM-Reboot
